packages/terra-clinical-detail-view/src/terra-dev-site/doc/example/_Detailview.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
Lets create a common sass file to share across the examples and lets drop the @extends syntax